terraform/workspaces/paragon/helm/storage.tf

```diff
@@ -0,0 +1,23 @@
+# EKS might create gp2 as a default. That is safe to unset with:
```

Comment:
AWS itself defaults to gp2. That is what the linked comment also states. I haven't found a way to have EKS create gp3 instead of gp2. The best I could do was to create an explicit gp3 storage class and make that the default. The alternative would be to create a gp3 storage class in each chart but since we use third party charts and also deploy to non-AWS environments the maintenance of that would outweigh this method.

Reply:
got it. ok thanks!!!  Still not a fan but I think there's enough context in the comment then for me to sign off.  just wanted to be 100% sure we're ok with this.